### PR TITLE
add init-bulk in dpgen documents

### PR DESCRIPTION
**Pull request by baihuyu12**

### CODE REVIEW

File: doc/init/init-bulk.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+## Init: Preparing Initial Data
```
Comment (njzjz) [inline doc/init/init-bulk.md:1]: TOC looks like a mass.
image

**Review by njzjz — CHANGES_REQUESTED** [review]

File: doc/init/init-bulk.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+| **pert_numb** | Integer | 30 | Number of perturbations for each POSCAR.
+| **pert_box** | Float | 0.03 | Percentage of Perturbation for cells.
+| **pert_atom** | Float | 0.01 | Perturbation of each atoms (Angstrom).
+| **md_nstep** | Integer | 10 | Steps of AIMD in stage 3. If it's not equal to settings via `NSW` in `md_incar`, DP-GEN will follow `NSW`.
+| **coll_ndata** | Integer | 5000 | Maximal number of collected data.
+| type_map | List | [ "Mg", "Al"] | The indices of elements in deepmd formats will be set in this order.
+| init_fp_style | String | "ABACUS" or "VASP" | First-principle software. If this key is absent, the default value will be "VASP".
+| relax_kpt | String | "....../KPT" | Path of `KPT` file for relaxation in stage 1. Only useful if `init_fp_style` is "ABACUS".
+| md_kpt | String | "....../KPT" | Path of `KPT` file for MD simulations in stage 3. Only useful if `init_fp_style` is "ABACUS".
+| atom_masses | List of float | [24] | List of atomic masses of elements. The order should be the same as `Elements`. Only useful if `init_fp_style` is "ABACUS".
```
Comment (njzjz) [inline doc/init/init-bulk.md:79]: This table should be generated automatically by `dargs`. But I don't request it done by this PR.
cc @HuangJiameng

**Review by njzjz — APPROVED** [review]

File: doc/init/init-bulk.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+```
+
+If you want to specify a structure as starting point for `init_bulk`, you may set in `PARAM` as follows.
+
+```json
+"from_poscar":	true,
+"from_poscar_path":	"....../C_mp-47_conventional.POSCAR",
+```
+`init_bulk` support both VASP and ABACUS for first-principle calculation. You can choose the software by specifying the key `init_fp_style`. If `init_fp_style` is not specified, the default software will be VASP.
```
Comment (njzjz) [inline doc/init/init-bulk.md:50]: support -> supports

**Review by AnguseZhang — APPROVED** [review]

**Review by njzjz — APPROVED** [review]

**Comment by AnguseZhang** [comment]
Screen Shot 2022-08-28 at 11 30 56 PM
Han and I both confronted such strange problem. We don't know how to solve it currently. Does it works fine in your place? @njzjz If so, please merge the PR directly. Otherwise, we may create a new PR.  Thanks.

**Comment by njzjz** [comment]
I have the same issue but I can merge it on app.

**Pull request merged**